orion/pkg/tests_integration/publish_integration_test.go

```diff
@@ -37,4 +36,225 @@ func Test_GetUnpublished(t *testing.T) {
 	assert.EqualValues(t, 2021, unpublishedDomains.Achieves[1].Year)
 	assert.EqualValues(t, "message2", unpublishedDomains.Achieves[1].Message)
 	assert.EqualValues(t, 2, len(unpublishedDomains.Achieves))
+
+	resetTable(t, domains.TABLE_ACHIEVEMENTS)
+}
+
+// Test: Get All Unpublished Programs, Publish a Few, then Get All Unpublished Again
+func Test_PublishPrograms(t *testing.T) {
```

Comment:
This endpoint is part of programs. `Test_GetUnpublished` should be the only test here.

Reply:
Not sure what you mean by that. GetUnpublished can only be meaningfully tested if we first publish some programs?

Reply:
Yes, but each domain has its own endpoint to publish, whereas the GetUnpublished is for all the domains.

Reply:
I get that, but testing GetUnpublished independently isn't very meaningful if we don't publish some of the domains and then see if the result changes. Then it would be no different than testing GetAll.

Reply:
Oh I see. You used both get all unpublished and get all published.